.prospector/opencraft.yml

```diff
@@ -31,7 +31,7 @@ profile-validator:
   options: {}
   run: null
 pyflakes:
-  disable: [F401, F403]
+  disable: [F401, F403, F841]
```

Comment:
This disables "pyflakes: F841 / local variable 'dummy' is assigned to but never used" since pylint has an equivalent check, and the pylint version supports inline disabling.